dist/t/spec/features/0030_project_spec.rb

```diff
@@ -22,6 +22,7 @@
       click_link('Your Home Project')
     end
     click_link('Repositories')
+    sleep(5)
```

Comment:
See [here](https://github.com/openSUSE/open-build-service/pull/15772/files#diff-5046c54ca51db7f1b66601407e62497dddd91065d2cba142ccb91dae3a8b6402R13) how it can be done more efficiently. Also, you can use [find](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Finders#find-instance_method) to check for the existence of some element and wait for it to appear until the default timeout kicks in.

Reply:
The tricky thing here is, that the "Leap 15.5" element is only displayed on the next page and I'm not even sure why waiting here helps. Maybe the backend syncs data over the interconnect from build.o.o?
So we would probably need to reload the 'Add from a Distribution' page until the checkbox appears.

Reply:
If "Leap 15.5" only is displayed after clicking on "Add from a Distribution", why don't you wait there?

Reply:
I tested that only waiting here helped. Maybe we could go to the next page, wait and then reload it, but I don't think that is better.